lib/plain/endpoints/editor-interface.ts

```diff
@@ -14,6 +13,13 @@ export const get = (http: AxiosInstance, params: GetEditorInterfaceParams) => {
   return raw.get<EditorInterfaceProps>(http, getBaseUrl(params))
 }
 
+export const getMany = (http: AxiosInstance, params: GetSpaceEnvironmentParams & QueryParams) => {
```

Comment:
The query params are not used, so I'd say either remove it from the signature or we actually use it.